code/modules/vehicle/ambulance.dm

```diff
@@ -14,20 +14,22 @@
 
 /obj/vehicle/ambulance/handle_vehicle_offsets()
 	..()
-	if(buckled_mob)
-		switch(buckled_mob.dir)
-			if(SOUTH)
-				buckled_mob.pixel_x = 0
-				buckled_mob.pixel_y = 7
-			if(WEST)
-				buckled_mob.pixel_x = 13
-				buckled_mob.pixel_y = 7
-			if(NORTH)
-				buckled_mob.pixel_x = 0
-				buckled_mob.pixel_y = 4
-			if(EAST)
-				buckled_mob.pixel_x = -13
-				buckled_mob.pixel_y = 7
+	if(buckled_mobs.len)
+		for(var/m in buckled_mobs)
```

Comment:
Change it to an iterative loop, like `for(var/i in 1 to buckled_mobs.len)`, and calculate the pixel offsets based on that so that mobs stack upwards